packages/server/utils/analytics/analytics.ts

```diff
@@ -320,6 +321,16 @@ class Analytics {
     this.track(userId, 'Summary Email Setting Changed', subscribeToSummaryEmail)
   }
 
+  accountCreated = (userId: string, isInvited: boolean, isPatient0: boolean) => {
+    this.track(userId, 'Account Created', {
+      isInvited,
+      // properties below needed for Google Analytics goal setting
+      category: 'All',
+      label: 'isPatient0',
+      value: isPatient0 ? 1 : 0
```

Comment:
You might notice that in the current version, this part of the code is:

```
      category: 'All',
      label: 'isInvited',
      value: isInvited ? 1 : 0
```

This has been overwritten instead of preserved because we are not actually using it anywhere.